types/ember__object/test/get-set.ts

```diff
@@ -0,0 +1,10 @@
+import { get, set } from '@ember/object';
+
+const basicPojo = { greeting: 'hello' };
+
+get(basicPojo, 'greeting'); // $ExpectType string
+set(basicPojo, 'greeting', 'ahoy'); // $ExpectType string
```

Comment:
And would be good to also have a test that attempting to set a bad key results in a type error:
```ts
set(basicPojo, 'salutation', 'ahoy'); // $ExpectError
```

Reply:
Ahh, good call.

Reply:
Actually: this is handled immediately below. 😂

Reply:
I swear that wasn't there a second ago!